### PR TITLE
Unify traces description

### PR DESCRIPTION
**Pull request by Ostrokrzew**
Change showing device's id to grouping statistics by device's description.
Fixes https://github.com/Open-CAS/standalone-linux-io-tracer/issues/166

Signed-off-by: Slawomir Jankowski <slawomir.jankowski@intel.com>

### CODE REVIEW

File: source/CMakeLists.txt [hunk 1]
```diff
@@ -156,4 +156,4 @@ file(APPEND ${CMAKE_BINARY_DIR}/octf-vars.cmake
 # Install it next to octf-config
 install(FILES ${CMAKE_BINARY_DIR}/octf-vars.cmake DESTINATION ${OCTF_PACKAGE_DIR} COMPONENT octf-install-cmake)
 
-add_subdirectory(examples)
+# add_subdirectory(examples)
```
Comment (mmkayPL) [inline source/CMakeLists.txt]: Revert

File: source/octf/analytics/statistics/FilesystemStatistics.cpp [hunk 2]
```diff
@@ -8,99 +8,96 @@
...
+        m_map.erase(key);
+
+        // Insert the new pair
+        auto result = m_map.emplace(pair);
+        if (!result.second || result.first == m_map.end()) {
+            throw Exception("Cannot allocate IO statistics");
+        }
+    } else {
+        // Key doesn't exist add it and allocate for them IO statistics
+        getStatisticsByKey(key);
```
Comment (mmkayPL) [inline source/octf/analytics/statistics/FilesystemStatistics.cpp]: I'd update the function to be more descriptive, or add the code here directly

File: source/octf/analytics/statistics/FilesystemStatistics.cpp [hunk 3]
```diff
@@ -8,99 +8,96 @@
...
 
     return *this;
 }
 
+void FilesystemStatistics::addDevice(
+        const proto::trace::EventDeviceDescription &devDesc) {
+    Key key(devDesc.id(), devDesc.size(), devDesc.name());
+
+    // Check maybe key already exists
+    auto iter = m_map.find(key);
```
Comment (mmkayPL) [inline source/octf/analytics/statistics/FilesystemStatistics.cpp]: In what situation the id would be the same and we'd want to reinsert the device with (I'm assuming) different size and name

**Comment by mariuszbarczak** [comment]
Need to be start over

**Pull request closed without merging**